lib/membrane/core/element/demand_handler.ex

```diff
@@ -30,7 +30,7 @@ defmodule Membrane.Core.Element.DemandHandler do
       `handle_delayed_demands`.
     * If element isn't supplying demand at the moment `handle_demand` is invoked right away, and it will
       invoke handle_demand callback, which will probably return :redemand and :buffers actions and in
```

Comment:
This was poorly written already, let's fix it

```suggestion
    * If the element isn't supplying demand at the moment and there's some unsupplied demand on the given 
      output, `handle_demand` is invoked right away, so that the demand can be synchronously supplied.
```